### ISSUE TITLE
Add New Prometheus System to monitor other service

### ISSUE THREAD

**Issue by xuanyuanaosheng** [kubesphere/kubesphere#3880]
Current the monitor system in kubesphere can not meet the monitoring requirements  due to the performance problems of Prometheus, So we need a new Prometheus to monitor other service ?

How to handle this ?

I test the 

```
apiVersion: monitoring.coreos.com/v1
kind: Prometheus
metadata:
  labels:
    prometheus: k8s-business
  name: k8s-business
  namespace: kubesphere-monitoring-system
spec:
  additionalScrapeConfigs:
    key: prometheus-additional.yaml
    name: additional-scrape-configs
  affinity:
    nodeAffinity:
      preferredDuringSchedulingIgnoredDuringExecution:
        - preference:
            matchExpressions:
              - key: node-role.kubernetes.io/monitoring
                operator: Exists
          weight: 100
  alerting:
    alertmanagers:
      - name: alertmanager-main
        namespace: kubesphere-monitoring-system
        port: web
  evaluationInterval: 5m
  image: 'prom/prometheus:v2.26.0'
  logLevel: debug
  nodeSelector:
    kubernetes.io/os: linux
  podMonitorNamespaceSelector:
    matchLabels:
      kubesphere.io/namespace: prom-exporter
  podMonitorSelector: {}
  probeNamespaceSelector:
    matchLabels:
      kubesphere.io/namespace: prom-exporter
  probeSelector: {}
  query:
    maxConcurrency: 1000
  replicas: 1
  resources:
    limits:
      cpu: '4'
      memory: 8Gi
    requests:
      cpu: 200m
      memory: 500Mi
  retention: 3d
  ruleSelector:
    matchLabels:
      prometheus: k8s-business
      role: alert-rules-business
  scrapeInterval: 1m
  secrets:
    - kube-etcd-client-certs
  securityContext:
    fsGroup: 0
    runAsNonRoot: false
    runAsUser: 0
  serviceAccountName: prometheus-k8s
  serviceMonitorNamespaceSelector:
    matchLabels:
      kubesphere.io/namespace: prom-exporter
  serviceMonitorSelector: {}
  storage:
    volumeClaimTemplate:
      spec:
        resources:
          requests:
            storage: 20Gi
  thanos:
    baseImage: quay.io/thanos/thanos
    version: v0.8.1
  tolerations:
    - effect: NoSchedule
      key: dedicated
      operator: Equal
      value: monitoring
  version: v2.26.0
```

The proemtheus works fine, but the kubesphere has error 
```
Internal Server Error
there is more than one prometheus custom resource in kubesphere-monitoring-system
```
![image](https://user-images.githubusercontent.com/4197714/118219103-60b7d600-b4ab-11eb-9994-9c9d55d7dea1.png)

**Comment by benjaminhuo** [comment]
@junotx Pls take a look at this issue. Is it possible to fix this in v3.1.1?

**Comment by junotx** [comment]
It is not supported to have multiple Prometheus instances in the `kubesphere-monitoring-system` namespace for now, and you can deploy new prometheus in another namespace

**Comment by junotx** [comment]
> @junotx Pls take a look at this issue. Is it possible to fix this in v3.1.1?

This is not a bug but the simultaneous support for multipe prometheus backends, which can lead to many changes, so it is not recommended to do so in v3.1.1

**Comment by zryfish** [comment]
> It is not supported to have multiple Prometheus instances in the `kubesphere-monitoring-system` namespace for kubesphere 3.1.0 alerting, and you can deploy new Prometheus in another namespace @xuanyuanaosheng

@xuanyuanaosheng  Does this answer your question?
